infra/terraform/terraform.go

```diff
@@ -250,6 +249,7 @@ func (r *terraform) boot() (output string, err error) {
 	out, err := r.command([]string{
 		"apply",
 		"-var", fmt.Sprintf("nodes=%d", r.NumNodes),
+		"-var", fmt.Sprintf("os=%s", r.OS),
```

Comment:
as discussed, TF destroy is not very reliable method of tearing down, and it'll be external service which would ensure cleanup by timeout. anyway right now destory is a no-op, and it'll be addressed in next few days.